public/assets/js/dashboard.js.LICENSE.txt

```diff
@@ -17,11 +17,6 @@ object-assign
  * @license  MIT
  */
 
-/*!
- * Font Awesome Free 5.15.1 by @fontawesome - https://fontawesome.com
- * License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
- */
-
```

Comment:
Where do these things come from?

Reply:
Not sure, and not sure why they've been deleted but will add back.